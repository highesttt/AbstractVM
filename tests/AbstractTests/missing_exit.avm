push int32(1)
dump