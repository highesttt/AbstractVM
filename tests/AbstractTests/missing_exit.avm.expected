Return: 84
Error: [1;33mMissing exit instruction
